tests/stress_tests/scripts/get_testdata.py

```diff
@@ -116,15 +117,19 @@ def main():
                         help='Skip errors caused by OMZ while downloading and converting.')
     args = parser.parse_args()
 
+    # constants
+    PRECISION = "FP32"
+
     # parse models from test config
-    tree = ET.parse(str(args.test_conf))
-    root = tree.getroot()
+    test_conf_obj = ET.parse(str(args.test_conf))
+    test_conf_root = test_conf_obj.getroot()
     models_from_cfg = []
-    for attributes in root:
-        if attributes.tag == "models":
-            models = [child.text for child in attributes]
-            models_from_cfg = [Path(model).stem for model in models]
-            break
+    for model_rec in test_conf_root.find("models"):
+        if model_rec.attrib.get("source") == "omz" and model_rec.attrib.get("name"):
+            models_from_cfg.append(model_rec.attrib["name"])
+        else:
+            log.warning("Record from test config '{}' doesn't contain attributes 'source' or 'name'"
```

Comment:
warning will be printed also if `model_rec.attrib["source"] !+= "omz"`. Need to change statement when warning should be printed

Reply:
Fixed